cmd/helm/init.go

```diff
@@ -390,7 +389,12 @@ func initLocalRepo(indexFile, cacheFile string, out io.Writer) (*repo.Entry, err
 		}
 
 		//TODO: take this out and replace with helm update functionality
-		os.Symlink(indexFile, cacheFile)
+		fp, err := filepath.Rel(filepath.Join(home.Cache(), cacheFile), indexFile)
+		if err != nil {
+			return nil, err
+		}
+		pth := home.Path(fp)
+		os.Symlink(pth, filepath.Join(home.Cache(), cacheFile))
```

Comment:
Hi,

os.Symlink does not work on Windows. Could you update to use createLink which calls the correct one depending on the OS?

Reply:
@pietromenna please find those changes already in my updated pr: https://github.com/kubernetes/helm/pull/3327, specifically the latest commit which adds an error return to createLink: https://github.com/kubernetes/helm/pull/3327/commits/15aee5d8f81c86e9b4869f061dd98e655a5442b8

Reply:
Cool! Thanks!

Reply:
@Stelminator Should I go ahead and close this PR in favor of yours? I don't want to two PRs open that are trying to solve same issue :)

Reply:
@rprobotics If you're ok with my PR, then, yes, please.  I think it's easier to maintain rebased w/o all the merge commits.